src/ui/src/builder/settings/BuilderFieldsWriterResourceId.vue

```diff
@@ -93,6 +94,24 @@ const selected = computed<string>({
 		setContentValue(component.value.id, fieldKey.value, String(value));
 	},
 });
+
+watch(
+	() => selected.value,
```

Comment:
Since `selected` is a reactive value (since it's `computed`), you can directly use 
```suggestion
	selected,
```